bin/radical-pilot-create-static-ve

```diff
@@ -232,16 +234,20 @@ then
     fi
 fi
 
+if test -z "$EDUMP"
+then
+    mkdir -p "$(pwd)/env"
+    EDUMP="$(pwd)/env/$(basename $PREFIX).env"
```

Comment:
@andre-merzky I'm not sure about this part - putting it into `env/` - should we keep as it is or set `EDUMP="$(pwd)/$(basename $PREFIX).env"`?

Reply:
I like that all envrinonment-related files end up in `env/`  - so the current works for me!